storybook/storybook-css/src/docs/stories.scss

```diff
@@ -0,0 +1,15 @@
+.amsterdam-docs-figure {
```

Comment:
Discuss what we should do with these styles.


The CSS for figures seemed necessary in one of the grid examples. A reset style sheet could take its place, and the future usage of our own components would obsolete them.

The pink highlighted box is useful to indicate otherwise invisible layout. Could still be useful, but it does clutter the examples. A preset in the toolbar for this story could have the same result, but I couldn’t find complete API docs for a story’s `parameters` very easily.

Reply:
Or perhaps a debug option for the grid that creates a pseudo element that is pink? I'm also curious to why NLDS doesn't contain a grid element.

Reply:
No-one implementing NLDS has started with an any specific layout as far as I know. Not sure why. It’s one of my focus points in design systems, so I guess we’ll be taking this for the team then.

A pink highlight will be useful in other layout components than grid alone, e.g. in our flex rows and columns, in size or aspect ratio components, or to document sizing design tokens.

I’ve created quite the arsenal of component specific for the design system docs in an earlier project, that’s why I’m kicking off with this approach here as well.

Instead of class names, I could also make a `<PinkBox>` component somewhere in a directory like `components/docs`. But we should probably check what Storybook offers us here first.

Reply:
Prefixing this with `amsterdam-` was actually a little confusing to me 😅 I thought it was styling that had to do with a component, and I didn't understand why it was in `/storybook`. Maybe just use `docs-`? And call the Sass file `docs.scss` as well?

Reply:
Yea something like that – the prefix was necessary because of an ESLint rule 😏